src/main/resources/assets/purpurclient/lang/nl_nl.json

```
@@ -0,0 +1,36 @@
+{
+  "purpurclient.options.title": "Purpur Client Opties",
+
+  "purpurclient.options.fix-chat-stutter": "Asynchrone Chat",
+  "purpurclient.options.fix-chat-stutter.tooltip" : "Inschakelen zal chat op een andere thread verwerken zodat de render thread niet blokkeert",
+  "purpurclient.options.bee-count-in-debug": "Bijen Telling",
+  "purpurclient.options.bee-count-in-debug.tooltip": "Inschakelen voegt de hoeveelheid bijen toe aan het debugscherm (F3) bij het bekijken van bijenkorven en nesten",
+  "purpurclient.options.splash-screen": "Opstartscherm",
+  "purpurclient.options.splash-screen.tooltip": "Inschakelen zal het Purpur opstartscherm gebruiken in plaats van de originele Mojang versie",
+  "purpurclient.options.window-title": "Venstertitel",
+  "purpurclient.options.window-title.tooltip": "Inschakelen zal het Purpur venstertitel gebruiken in plaats van de originele Mojang titel",
+  "purpurclient.options.mobs": "Mobs",
+  "purpurclient.options.mobs.tooltip": "Opent het mob opties scherm",
+
+  "purpurclient.title.singleplayer": "%s in Singleplayer",
+  "purpurclient.title.multiplayer.realms": "Multiplayer (%s op Realms)",
+  "purpurclient.title.multiplayer.lan": "Multiplayer (%s in LAN)",
+  "purpurclient.title.multiplayer.server": "Multiplayer (%s op %s)",
```

Mojang translates singleplayer and multiplayer to `alleen spelen & samen spelen`
```suggestion
  "purpurclient.title.multiplayer.server": "Samen spelen (%s op %s)",
```